lib/js/components/TabItem/TabItem.vue

```diff
@@ -49,18 +94,24 @@ export default {
 				return Object.values(ICONS).includes(toRaw(icon));
 			},
 		},
-		isActive: {
+		isSelected: {
 			type: Boolean,
 			required: true,
 		},
-		title: {
+		label: {
 			type: String,
 			required: true,
 		},
+		size: {
+			type: String,
+			default: TAB_ITEM_SIZES.MEDIUM,
+		},
 	},
 	data() {
 		return {
-			ICON_SIZES: Object.freeze(ICON_SIZES),
+			iconSize:
```

Comment:
this should be computed. What if I change the size on a instance that is already mounted?